src/librustc/session/config.rs

```diff
@@ -1052,7 +1052,7 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
     linker_flavor: Option<LinkerFlavor> = (None, parse_linker_flavor, [UNTRACKED],
                                            "Linker flavor"),
     fuel: Option<(String, u64)> = (None, parse_optimization_fuel, [TRACKED],
-        "Set the optimization fuel quota for a crate."),
```

Comment:
This is one of two options that is capitalized. All the others start with lowercase letters.